### ISSUE TITLE
separate input to input files and folders

### ISSUE THREAD

**Issue by yucongalicechen** [diffpy/diffpy.labpdfproc#131]
### Problem

CLI takes input files and folders in one argument, but GUI only supports widget for selecting files or folders. Currently the CLI input for GUI is implemented on selecting files. See #130 for more details.

### Proposed solution
Separate input to files and folders to match the functionalities in GUI.

**Comment by yucongalicechen** [comment]
@sbillinge I'm revisiting this issue. Previously we considered separating `input` to `input files` and `input folders` in gui because it doesn't support widget of selecting multiple files and folders at the same time. 
I found that gui also doesn't support multiple folders. Also, since input is a required argument in our CLI, separating it into two arguments would make the requirement on one of the more complicated.
Shall we just allow selecting multiple files for gui? If users need to add a folder, they can either select all files inside it or provide the file path in a text file.

**Comment by sbillinge** [comment]
yes, that is a good solution.  In general, gui's will allow you to CTL-A to select all so it is easy to select everything in a folder.

If we want to have the CLI functionality of globbing a full directory tree, we could allow input to be "glob" or something and then allow the user to set an optional input which would be the path to the top-level directory and the glob pattern.

This may be confusing to non-programmers, so we could hide the functionality a bit, buried in the instructions.

For now, maybe just capture this in an issue that we can backburner for the future if it is asked for, and do it the way you suggest.

**Comment by yucongalicechen** [comment]
Yeah I think we already have glob functionality in CLI. I'll keep this issue open then and clarify about the widget either in docs or through help messages.

**Comment by sbillinge** [comment]
> Yeah I think we already have glob functionality in CLI. I'll keep this issue open then and clarify about the widget either in docs or through help messages.

yes, this is very useful for our more pythony command line folks, so it would be sad to lose it completely if we can figure out how to handle it in the gui (which is for our non-pythony users)
